pkg/graveler/graveler_test.go

```diff
@@ -2292,3 +2292,182 @@ func TestGravelerCreateCommitRecord(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestGraveler_Revert(t *testing.T) {
+	type fields struct {
+		CommittedManager *testutil.CommittedFake
+		RefManager       *testutil.RefsFake
+	}
+	type args struct {
+		ctx          context.Context
+		branchID     graveler.BranchID
+		ref          graveler.Ref
+		parentNumber int
+		allowEmpty   bool
+	}
+	tests := []struct {
+		name        string
+		fields      fields
+		args        args
```

Comment:
This:
```suggestion
		revertArgs        args
```
might make it clear why we have `fields` and `args`.